src/app_service/service/chat/service.nim

```diff
@@ -573,13 +573,13 @@ QtObject:
     except Exception as e:
       error "Error requesting transaction", msg = e.msg
 
-  proc muteChat*(self: Service, chatId: string) =
+  proc muteChat*(self: Service, chatId: string, interval: int) =
     try:
       if(chatId.len == 0):
         error "error trying to mute chat with an empty id"
         return
 
-      let response = status_chat.muteChat(chatId)
+      let response = status_chat.muteChat(chatId, interval)
       if(not response.error.isNil):
         let msg = response.error.message & " chatId=" & chatId
         error "error while mute chat ", msg
```

Comment:
I can't comment below here, but do you think we need to pass the interval to the event? We'll probably need to update the chat. Might be worth using the response from the RPC call for that.
Do you plan on doing that in the next PR?

Reply:
oh about event I think not, it works fine I tested it. Also, I don't see any UI changes in Sigma about unmuting menu item or displaying an interval there, so we need just fact of muting. And then we should have an update event about `muted: false`

Reply:
Makes sense.

I guess the only thing is to update the chat cache. 

I assume `response` contains `chat` and you can call `updateChat` with that chat to update the cache

Reply:
it's works as works before :)

Reply:
Anyway, we can merge it cos I'm planing to make final testing when I implement it for communities

Reply:
> it's works as works before :)

Yeah it probably does. It's a good opportunity to fix the cache inconsistency though. It's something that might have bit us later.

> Anyway, we can merge it cos I'm planing to make final testing when I implement it for communities

Sure, but please do it then. It should be quite simple if the chat is already part of the response.